src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public static Ball getRandomBall() {
+        int randomNumber = new Random().nextInt(100);
+        Ball ball = new Ball(getRandomColor(), randomNumber);
+        return ball;
+    }
```

Comment:
1. Lottery shouldn't extend ColorSupplier, it depends on it
2. Magic numbers should be extracted to constants with proper naming
3. Services we rely on are better as class variables in order not to create new object each time method is called
4. Do not use static methods unless it's really necessary
```suggestion
public class Lottery {
    private static final int MAX_BALL_NUMBER = 100;

    private final Random random = new Random();
    private final ColorSupplier colorSupplier = new ColorSupplier();

    public Ball getRandomBall() {
        int randomNumber = random.nextInt(MAX_BALL_NUMBER);
        Ball ball = new Ball(colorSupplier.getRandomColor(), randomNumber);
        return ball;
    }
```